### PR TITLE
tests(ssim): fix the randomly failing test

### LINKED ISSUE #118: Randomly failing SSIM test
**Describe the bug**
Test `test_multi_scale_ssim_loss_raise_if_wrong_value_is_estimated_custom_weights` randomly fails.

**To Reproduce**
1. Run tests in you local environment of on GitHub with some [non-related changes](https://github.com/photosynthesis-team/piq/pull/117)
2. Observe the [test failing](https://github.com/photosynthesis-team/piq/runs/819666366) from time to time

**Expected behavior**
Test fails if and only if the implementation of tested functionality is incorrect

**Screenshots**
image

### PR DESCRIPTION
**Pull request by snk4tr**
Closes #118 

## Proposed Changes

  - Decrease match accuracy between TF and our implementation by an order of magnitude

It was tested that the proposed change helps by running the test 1k times:
```python
import torch
import tensorflow as tf

from tqdm import tqdm

from tests.test_ssim import test_multi_scale_ssim_loss_raise_if_wrong_value_is_estimated_custom_weights


def main():
    tf.enable_eager_execution()
    prediction = torch.rand(3, 3, 256, 256)
    target = torch.rand(3, 3, 256, 256)
    for _ in tqdm(range(1000)):
        test_multi_scale_ssim_loss_raise_if_wrong_value_is_estimated_custom_weights(prediction, target)


if __name__ == '__main__':
    main()
```

### CODE REVIEW

**Comment by denproc** [comment]
I did some changes to your script and initialised the prediction/target at each iteration.
I tested the mentioned functions for 1000 iterations:
❌ `test_multi_scale_ssim_raise_if_wrong_value_is_estimated`

❌ `test_multi_scale_ssim_raise_if_wrong_value_is_estimated_custom_weights`

✅ `test_multi_scale_ssim_loss_raise_if_wrong_value_is_estimated`

✅ `test_multi_scale_ssim_loss_raise_if_wrong_value_is_estimated_custom_weights`

✅ `test_brisque_values_grey`

❌ `test_brisque_values_RGB`

Therefor, I suggest to take a closer look to the implementation to increase the precision. I'll change the existing issue and reassign to myself.

**Comment by snk4tr** [comment]
> I did some changes to your script and initialised the prediction/target at each iteration.
> I tested the mentioned functions for 1000 iterations:
> ❌ `test_multi_scale_ssim_raise_if_wrong_value_is_estimated`
> 
> ❌ `test_multi_scale_ssim_raise_if_wrong_value_is_estimated_custom_weights`
> 
> ✅ `test_multi_scale_ssim_loss_raise_if_wrong_value_is_estimated`
> 
> ✅ `test_multi_scale_ssim_loss_raise_if_wrong_value_is_estimated_custom_weights`
> 
> ✅ `test_brisque_values_grey`
> 
> ❌ `test_brisque_values_RGB`
> 
> Therefor, I suggest to take a closer look to the implementation to increase the precision. I'll change the existing issue and reassign to myself.

Thanks for checking. Closing this one then.
